lumicks/pylake/tests/test_imaging_camera/test_correlated_stack.py

```diff
@@ -531,3 +532,118 @@ def check_result(stack, ref_stack, bit_depth):
     offset_correct_points = [np.array(pp) - (25, 25) for pp in correct_points]
     stack = stack.define_tether(*offset_correct_points)
     check_result(stack, ref_stack, 16)
+
+
+@cleanup
```

Comment:
how would you feel about pulling all the plotting tests out into a separate file? I think there's enough that it makes sense, and this one is getting pretty long

If you don't wanna piggyback it on this PR that's fine, but I think it's something we should do in general for all the image class tests

Reply:
I can do a PR for that immediately after this one. The reason I'd prefer to not do it immediately is to not get into a mess with fixups and moves at the same time.